test/unit/mock-transform.spec.ts

```diff
@@ -77,10 +77,38 @@ describe('mocking', () => {
         from: 'bob',
       }]
       const code = await getResult(`
-        import { expect, vi } from 'vitest'
+        import { expect, vi, it } from 'vitest'
         mockNuxtImport('useSomeExport', () => 'bob')
+        
+        it('test', () => {
+          const a = vi.fn()
+        })
+      `)
+      expect(code).toMatchInlineSnapshot(`
+        "import {vi} from "vitest";
+        vi.hoisted(() => { 
+                if(!globalThis.__NUXT_VITEST_MOCKS){
+                  vi.stubGlobal("__NUXT_VITEST_MOCKS", {})
+                }
+              });
+
+                import { expect, vi, it } from 'vitest'vi.mock("bob", async (importOriginal) => {
```

Comment:
should we remove the later import?

Reply:
we can move this one up instead of creating a new one

Reply:
I think the first import is just to satisfy vitest analysis. It should not have an actual effect.

Reply:
This is just guessing:

Maybe the transform is too late for vitest

https://github.com/nuxt/test-utils/blob/main/src/module/plugins/mock.ts#L255

Reply:
maybe just reorder it to beginning of file ....

Reply:
actually, I think I've found the issue - give me a sec